### PR TITLE
Null test fixes

### PR DESCRIPTION
**Pull request by losborne**
This PR updates tests so that they assert when things should be null with `should(...).be.null()`.  Previously, `should(...).be.null` would return `undefined`, not assert (thus not fail), and the tests would silently pass.

After changing all the `null`s to `null()`s, there were a couple of failing tests, which are skipped below because they do not match the current cql spec.

(NOTE: I tested cherry picking these 2 commits into https://github.com/cqframework/cql-execution/pull/31 and tests still pass for that PR.)

Pull requests into cql-execution require the following.
Submitter and reviewer should ✔ when done.
For items that are not-applicable, mark "N/A" and ✔.

[CDS Connect](https://cds.ahrq.gov/cdsconnect) and [Bonnie](https://github.com/projecttacoma/bonnie) are the main users of this repository. 
It is strongly recommended to include a person from each of those projects as a reviewer.

**Submitter:**
- [x] This pull request describes why these changes were made
- [x] Code diff has been done and been reviewed (it does not contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] Code coverage has not gone down and all code touched or added is covered.
- [x] All dependent libraries are appropriately updated or have a corresponding PR related to this change N/A only update to tests

**Reviewer:**

Name: @cmoesel 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer:**

Name: @edeyoung 
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by losborne** [comment]
@cmoesel updated and force pushed to match latest master

File: test/elm/list/test.coffee [hunk 1]
```diff
@@ -289,9 +289,13 @@ describe 'Includes', ->
...
-  it 'should return null if either arg is null', ->
-    should(@nullIncluded.exec(@ctx)).be.null()
-    should(@nullIncludes.exec(@ctx)).be.null()
+  it.skip 'should return true if right arg is null', ->
+    # TODO: currently returns false
+    should(@nullIncluded.exec(@ctx)).be.true()
+
+  it.skip 'should return false if left arg is null', ->
+    # TODO: currently returns null
+    should(@nullIncludes.exec(@ctx)).be.false()
```
Comment (cmoesel) [inline test/elm/list/test.coffee]: To ensure we're on the same page... you're skipping these because the current code actually produces the *wrong* answers and we don't want tests that assert them as being correct when they actually aren't?  And fixing the logic and re-enabling the tests would go into a different PR?

Comment (losborne) [reply test/elm/list/test.coffee]: @cmoesel Correct. The new expected values match HL7_CQLANG_R1_STU2_2017JUL.  

List Includes:
image
List IncludedIn:
image

Even if we fixed them to match the spec, those tests are currently being translated directly to `contains` and `In`, which have different behavior than `includes` and `includedIn` after the most recent errata to the spec.

For now I think it's best to skip these tests with the expected behavior matching the spec, then it's a future task to figure out what to do about the discrepancies (if anything).

Comment (cmoesel) [reply test/elm/list/test.coffee]: OK.  Sounds good to me.

**Comment by cmoesel** [comment]
Do you need a second reviewer before I can merge?  It seems to me that since this only fixes test logic, and has no affect on the packaged/production code, maybe one review is enough?

**Comment by losborne** [comment]
@cmoesel Yes good point--one reviewer is fine for this one.  Merge away!
